src/lib/colors.js

```diff
@@ -10,6 +10,7 @@ import { hsl } from 'd3-color';
 const colors = {
   primaryColor: '#79B63E',
   primaryColorDarker: '#4d7227',
+  primaryColorBrighter: 'rgba(0, 0, 0, 0.7)',
```

Comment:
Wouldn't `primaryColorBrighter` mean that this is green? I guess the code should use `textColor` instead.

Reply:
@nero-still colors!